### ISSUE TITLE
[Bug] Decks.select crash

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#10046]
User reported, have a collection with the issue

```
java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.ichi2.utils.JSONObject.getString(java.lang.String)' on a null object reference
at com.ichi2.libanki.Decks.select(Decks.java:1)
at com.ichi2.libanki.Decks.current(Decks.java:2)
at com.ichi2.anki.DeckPicker.__renderPage(DeckPicker.java:32)
at com.ichi2.anki.DeckPicker$UpdateDeckListListener.actualOnPostExecute(DeckPicker.java:9)
at com.ichi2.anki.DeckPicker$UpdateDeckListListener.actualOnPostExecute(DeckPicker.java:1)
at com.ichi2.async.TaskListenerWithContext.onPostExecute(TaskListenerWithContext.java:2)
at com.ichi2.async.CollectionTask.onPostExecute(CollectionTask.java:3)
at android.os.AsyncTask.finish(AsyncTask.java:771)
at android.os.AsyncTask.access$900(AsyncTask.java:199)
at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:788)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loop(Looper.java:246)
at android.app.ActivityThread.main(ActivityThread.java:8506)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130) 
```

2.15 and latest alpha

**Comment by david-allison-1** [comment]
The provided collection only has `conf` defined. But this causes a crash so the user cannot change the Advanced settings

**Comment by s-deepak-kumar** [comment]
Can You Please Assign It To Me?

**Comment by dae** [comment]
Presumably closed by #14171.
